tests/client/ui/components/TestCard.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { renderIntoDocument } from 'react-addons-test-utils';
+
+import Card from 'ui/components/Card';
+
+
+describe('<Card />', () => {
+  function render(props = {}) {
+    return renderIntoDocument(<Card {...props} />);
+  }
+
+  it('renders a Card', () => {
+    const root = render({ className: 'TofuSection' });
+    assert(root.cardContainer);
+    assert.equal(root.cardContainer.tagName, 'SECTION');
+    assert.include(root.cardContainer.className, 'Card');
+    assert.include(root.cardContainer.className, 'TofuSection');
+  });
+
+  it('shows header if supplied', () => {
+    const root = render({ header: 'foo' });
+    assert(root.header);
+  });
+
+  it('hides header if none supplied', () => {
+    const root = render({ children: 'hello' });
+    assert(!root.header);
+    assert.include(root.contents.className, 'no-header');
```

Comment:
These classes should be prefixed as `Card--no-header`, and they might make more sense on `.Card` than `.Card-contents` since you can easily select `.Card--no-header .Card-contents`.